code-of-conduct-IT.md

```diff
@@ -0,0 +1,42 @@
+## Introduzione
+
+Ci impegniamo a garantire a tutti un'esperienza senza molestie e non le tolleriamo in nessuna forma. Ti chiediamo di tenere in considerazione gli altri e di comportarti professionalmente e con rispetto verso tutti gli altri partecipanti. Questo codice e le relative pratiche si applicano anche ai comportamenti inaccettabili che si verificano al di fuori dell'ambito delle attività comunitarie, in qualsiasi luogo della comunità - in linea e di persona - così come in tutte le comunicazioni dirette e dovunque tale comportamento possa avere influenzare negativamente la sicurezza e il benessere dei membri della comunità. Gli espositori, oratori, sponsor, il personale e tutti gli altri partecipanti agli eventi organizzati da Docker, Inc (DockerCon, meetups, riunioni di utenti) o presso le strutture sono soggetti a queste linee guida comunitarie e codice di condotta.
+
+La diversità e l'accettazione rendono forte la comunità di Docker. Noi incoraggiamo la partecipazione di persone dalle molte esperienze diverse e vogliamo che la nostra posizione sia molto chiara.
```

Comment:
questa non la condivido. Leggendo il documento io capisco che incoraggiando la partecipazioni di persone con esperienze diverse vogliono che la loro posizione (where we stand) riguardo le discriminazioni ecc sia chiara. E non che per dimostrare di essere aperti incoraggiano le persone...
A parte questo condivido che il paragrafo vada migliorato e l'ho cambiata in 

> La diversità e l'accettazione rendono forte la comunità di Docker e, incoraggiando la partecipazione di persone con esperienze molto diverse, vogliamo che la nostra posizione sia molto chiara.

>